test/JDBC/expected/exec_sp_in_udf-vu-prepare.out

```diff
@@ -0,0 +1,32 @@
+use master
+go
+create table t1_exec_sp_in_udf (a int)
+go
+create procedure sp_myproc as return 0
+go
+create function f_scalar_exec_sp_in_udf(@p int) 
```

Comment:
Ideally we would also want MVU tests where functions have already been created using sp_ procedures in the body in previous versions (prepare script) and then verified on latest

Reply:
In some previous releases, this could crash the PG server, so avoiding including that in the tests.